router/pkg/routerconfig/s3/client.go

```diff
@@ -30,13 +31,29 @@ type ClientOptions struct {
 }
 
 func NewClient(endpoint string, options *ClientOptions) (routerconfig.Client, error) {
-
 	client := &Client{
 		options: options,
 	}
 
+	// The providers credential chain is used to allow multiple authentication methods.
+	providers := []credentials.Provider{
+		// Static credentials allow setting the access key and secret access key directly.
+		&credentials.Static{
+			Value: credentials.Value{
```

Comment:
According to minio this would not produce the same config as we did with `NewStaticV4`. You would need to add `	SignerType:      credentials.SignatureV4`

Reply:
That's done now https://github.com/wundergraph/cosmo/pull/1250/commits/48e2a1be710d6edb9731a685f1e7c6a13f8e980c